lib/iris/experimental/um.py

```diff
@@ -726,20 +725,43 @@ def cutout(ffv_src, filename, cutout_params):
 
     * ffv_src (:class:`FieldsFileVariant`)
         A source FieldFileVariant to subset.
-    * filename (string)
-        The name of the file containing the subset of the source UM
-        FieldsFile variant.
+    * filepath (string)
```

Comment:
Ok, it doesn't match the usage in FieldsFileVariant itself, which is confusing, so I'll change it back...